script.js

```diff
@@ -1,4 +1,128 @@
+// gamemode , enter username name
+var gameMode = 'enter username mode';
+
+// store username
+var userNameOfThePlayer = '';
+
+// random number from 0 to 2
+var randomNum = function () {
+  var numZeroToTwo = Math.floor(Math.random() * 3);
+  return numZeroToTwo;
+};
+
+// user win
+var numOfUserWin = 0;
+
+// computer win
+var numOfComputerWin = 0;
+
+// total game
+var totalMatch = 0;
+
+// winning percentage
+var userWinPercentage = 0;
+var computerWinPercentage = 0;
+
+// number of draw
```

Comment:
This is a useless comment. Please use comments to explain more about why a variable exists instead of just repeating the variable name.